packages/ingest/S3ProviderClient.js

```diff
@@ -5,7 +5,12 @@ const log = require('@cumulus/common/log');
 const path = require('path');
 const errors = require('@cumulus/common/errors');
 
-module.exports.s3Mixin = (superclass) => class extends superclass {
+class S3ProviderClient {
+  constructor(providerConfig) {
+    this.bucket = providerConfig.bucket;
```

Comment:
Should we be validating/throwing errors if these properties are not provided on the `providerConfig` input?